susemanager-utils/susemanager-sls/src/modules/kiwi_info.py

```diff
@@ -6,6 +6,13 @@
 
 log = logging.getLogger(__name__)
 
+# Kiwi version is always in format <MAJOR:MINOR:PATCH> with numeric values
+KIWI_VERSION_REGEX=r'\d+\.\d+\.\d+'
+# Taken from Kiwi sources https://github.com/OSInside/kiwi/blob/master/kiwi/schema/kiwi.rng#L81
```

Comment:
Could we turn this into a permalink? Links to line numbers aren't really stable over time when they link a moving target such as a branch.